### ISSUE TITLE
ImageNet-C dataset 

### ISSUE THREAD

**Issue by waychin-weiqin** [mariodoebler/test-time-adaptation#15]
Hi, 

I am wondering what is the reason behind using only 5000 samples from ImageNet-C rather than using all 50000 samples? 

Thank you.

**Comment by RobMarsden** [comment]
Hi, 
Thanks for the question. CoTTA introduced the 5000 samples setting by simply following the default setting provided by robustbench ([see here](https://github.com/qinenergy/cotta/tree/main/imagenet/robustbench/data)). We support both settings, so researchers can decide which setting they want to study. To use all 50k samples, just specify `CORRUPTION.NUM_EX 50000`

**Comment by waychin-weiqin** [comment]
Thanks a bunch for the reply! It is really helpful. 

I am wondering if you could also provide the hyperparameters (e.g. batch size, learning rate, optimizer, etc) for TENT, ETA and EATA for ViT / Swin models? 

Look forward to your reply! Thank you :)

**Comment by RobMarsden** [comment]
We used the same cfg files and simply replaced the model by setting `MODEL.ARCH vit_b_16`, for example. If you want to reproduce the results in [our paper](https://arxiv.org/abs/2306.00650), you can also check out the [run.sh](https://github.com/mariodoebler/test-time-adaptation/blob/main/classification/scripts/run.sh) located in the `classification/scripts` directory.
Best regards,
Rob
